src/pages/iou/request/step/IOURequestEditReportCommon.tsx

```diff
@@ -87,20 +87,22 @@ function IOURequestEditReportCommon({
             return [];
         }
 
-        return Object.values(allPoliciesID ?? {}).flatMap((policyID) => {
-            if (!policyID || (policyIDFromProps && policyID !== policyIDFromProps)) {
-                return [];
-            }
-            const reports = getOutstandingReportsForUser(
-                policyID,
-                reportOwnerAccountID,
-                outstandingReportsByPolicyID?.[policyID ?? CONST.DEFAULT_NUMBER_ID] ?? {},
-                reportNameValuePairs,
-                isEditing,
-            );
-
-            return reports;
-        });
+        return Object.values(allPoliciesID ?? {})
+            .filter((policyID) => getPersonalPolicy()?.id !== policyID)
```

Comment:
Let's move the call to getPersonalPolicy() outside of this loop since it won't change and we can avoid calling getPersonalPolicy() for each policyID